packages/artillery-plugin-expect/index.js

```diff
@@ -83,10 +83,13 @@ function expectationsPluginOnError(
   events,
   done
 ) {
-  if (userContext.expectationsPlugin.outputFormat === 'json') {
-    artillery.log(JSON.stringify({ ok: false, error: scenarioErr.message }));
+  if (scenarioErr.message.startsWith('Failed expectations for request')) {
+    return done();
+  }
+  if (userContext.expectationsPlugin.formatter === 'json') {
+    artillery.log(JSON.stringify({ error: scenarioErr.message }));
```

Comment:
```suggestion
    artillery.log(JSON.stringify({ ok: false, error: scenarioErr.message }));
```

Best not to remove fields someone might rely on without a good reason.